src/circuits/notes/note.tcc

```diff
@@ -5,6 +5,8 @@
 // Content Taken and adapted from Zcash
 // https://github.com/zcash/zcash/blob/master/src/zcash/circuit/note.tcc
 
+#include "note.hpp"
```

Comment:
The tcc is already included in the hpp

Reply:
Yes, technically it's redundant, but it allows the file to be opened in an IDE with all symbols in scope.

Reply:
Ah I see. We can keep it that way then if it is easier for you; the preprocessor conditional inclusion directives (#ifdef ...) should take care of avoiding cycling dependencies